scripts/install-snapshot.sh

```diff
@@ -84,30 +84,6 @@ function delete_snapshot_crd() {
     kubectl delete -f "${VOLUME_SNAPSHOT}" --ignore-not-found
 }
 
```

Comment:
Typo in the commit message, not version 1.7 but 1.17.

Reply:
corrected the same. thanks :+1: